main.cpp

```diff
@@ -485,6 +486,21 @@ void ChannelStrip::acceptAudioInput(bool isConnected, bool isMono)
     std::cout << "How loud can it go? " << maxGainRange << "db" << std::endl;
 }
 
+float ChannelStrip::addChannel(int numChannels)
+{
+    for (int i = 0; i < numChannels; i++)
+    {
+        int channel = 0;
+
+        if (channel > 1)
+        {
+            numChannels = channel += 1;
+            return numChannels;
+        }
+    }
+    return numChannels;
```

Comment:
you have 2 `return numChannels;` here.
`break;` out of your `for()` loop so you only have 1 `return` statement at the end.
- [ ] revise

Reply:
Ah, I didn't know about this.